lib/net.c

```diff
@@ -67,9 +62,18 @@ close_num_ports(uint8_t nb_ports)
 	}
 }
 
+struct net_config *
+alloc_net_conf(void)
+{
+	config.gatekeeper_pktmbuf_pool = (struct rte_mempool **) \
```

Comment:
Given that the number of entries in the array is constant, why not making it part of the definition of the struct? If dynamical allocation is needed, there should be a test to make sure it wasn't already allocated. Otherwise, calling `alloc_net_conf()` twice would lose the first reference.

To make clear that there is only a `net_conf` variable in Gatekeeper, the function should be called `get_net_conf()` instead of `alloc_net_conf()`.